internal/storage/pebbledb.go

```diff
@@ -19,7 +19,6 @@ var _ DB = (*PebbleDB)(nil)
 // NewPebbleDB returns a new PebbleDB instance using the default options.
 func NewPebbleDB(name, dir string) (*PebbleDB, error) {
 	opts := &pebble.Options{}
-	opts.EnsureDefaults()
```

Comment:
not needed?

Reply:
`NewPebbleDB` always calls `NewPebbleDBWithOpts` which already calls `opts.EnsureDefaults()` 🙂